go.mod

```diff
@@ -3,11 +3,17 @@ module github.com/IBM-Cloud/power-go-client
 go 1.13
 
 require (
-	github.com/IBM/go-sdk-core/v5 v5.8.0
+	github.com/IBM/go-sdk-core/v5 v5.9.1
 	github.com/apparentlymart/go-cidr v1.1.0
+	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d // indirect
```

Comment:
Did you run `go mod tidy`?

Reply:
Running `go mod tidy` did not update this at all.